src/caret_analyze/infra/lttng/records_provider_lttng.py

```diff
@@ -666,6 +740,27 @@ def get_qos(
         return self._lttng.get_publisher_qos(pubs_lttng[0])
 
     def get_sim_time_converter(self, min_ns, max_ns) -> ClockConverter:
+        """
+        Get sim time converter.
+
+        Parameters
+        ----------
+        min_ns : float
+            Min time.
+        max_ns : float
+            Max time.
+
+        Returns
+        -------
+        ClockConverter
+            Clock converter
+
+        Raises
+        ------
+        InvalidArgumentError
+            Failed to load sim_time.
+
+        """
```

Comment:
I have added a few explanations about the meaning of each argument etc.
```suggestion
        """
        Get sim time converter.

        Parameters
        ----------
        min_ns : float
            Minimum timestamp value of the data used to create the system time to sim_time converter.
        max_ns : float
            Maximum timestamp value of the data used to create the system time to sim_time converter.

        Returns
        -------
        ClockConverter
            Object that converts timestamps from system time to sim_time.

        Raises
        ------
        InvalidArgumentError
            Failed to load sim_time.

        """
```

Reply:
Fixed